src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -1658,6 +1658,24 @@ bool SQLPTRepresentation::GatherModuleType(
   return true;
 }
 
+bool SQLPTRepresentation::SaveRemoteControlDenied(const std::string& app_id,
+                                                  bool deny) {
```

Comment:
@dev-gh it was accidentally deleted in 
bca7dd9f538fba067212d52e4ae94d6edd732be5
as I understood.
And we use it in SQLPTRepresentation::SaveSpecificAppPolicy function